src/styles/_catalog.scss

```diff
@@ -13,10 +27,5 @@
     height: 300px;
```

Comment:
can't comment on lines that weren't changed - why do you need width here? it should be controlled by grid